meson.build

```diff
@@ -2,15 +2,25 @@ project('dosbox-staging', 'c', 'cpp',
         default_options : ['cpp_std=c++14', 'b_ndebug=if-release'],
         version : '0.77.0')
 
-assert(meson.version().version_compare('>= 0.45.1'),
-       'Expecting meson 0.45.1 or newer.')
+
+# meson version check
+#
+# After rising the oldest supported meson version, make the following
+# improvements:
```

Comment:
After increasing the minimum-required meson version, make the following improvements: